api/geth_backend.go

```diff
@@ -1058,6 +1061,7 @@ func (b *GethStatusBackend) StartNodeWithAccountAndInitialConfig(
 	return b.StartNodeWithAccount(account, password, nodecfg)
 }
 
+// TODO: change in `saveAccountsAndSettings` function param `subaccs []*accounts.Account` parameter to `profileKeypair *accounts.Keypair` parameter
```

Comment:
won't include this change in this PR?

Reply:
Isn't already too much for this PR? Thought to add this to another one, cause that change will require more changes here and as well more changes on the client side.